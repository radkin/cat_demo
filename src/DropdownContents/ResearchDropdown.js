import React from "react"
import styled from "styled-components"
import {
  Heading,
  HeadingLink,
  LinkList,
  DropdownSection,
  Icon
} from "./Components"

const ResearchDropdownEl = styled.div`
  width: 18.5rem;
`

const ResearchDropdown = () => {
  return (
    <ResearchDropdownEl>
      <DropdownSection data-first-dropdown-section>
        <ul>
          <HeadingLink>
            <a
              target="_blank"
              rel="noopener noreferrer"
              href="https://fringe.fandom.com/wiki/Synaptic_Transfer">
              <Icon />Synaptic Transfer
            </a>
          </HeadingLink>
          <HeadingLink>
            <a
              target="_blank"
              rel="noopener noreferrer"
              href="https://fringe.fandom.com/wiki/Rapid_skin_growth_toxin">
              <Icon />Rapid Skin Growth Toxin
            </a>
          </HeadingLink>
          <HeadingLink>
            <a
              target="_blank"
              rel="noopener noreferrer"
              href="https://www.popularmechanics.com/culture/tv/a5477/4344500/">
              <Icon />Targeted Genetics
            </a>
          </HeadingLink>
          <HeadingLink noMarginBottom>
            <a
              target="_blank"
              rel="noopener noreferrer"
              href="https://fringe.fandom.com/wiki/Alternate_Universe">
              <Icon />Multidimentional Travel
            </a>
          </HeadingLink>
        </ul>
      </DropdownSection>
      <DropdownSection>
        <div>
          <Heading>
            <Icon />Interactive
          </Heading>
          <LinkList marginLeft="25px">
            <li>
              <a
                target="_blank"
                rel="noopener noreferrer"
                href="https://fringe.fandom.com/wiki/The_Zodiac_Paradox">
                Zodiac Paradox &rsaquo;
                </a>
            </li>
            <li>
              <a
                target="_blank"
                rel="noopener noreferrer"
                href="https://en.wikipedia.org/wiki/Transhumanism">
                Enhance physiology and intellect &rsaquo;
              </a>
            </li>
            <li>
              <a
                target="_blank"
                rel="noopener noreferrer"
                href="https://en.wikipedia.org/wiki/Technological_singularity">
                Unforeseeable changes to human civilization &rsaquo;</a>
            </li>
          </LinkList>
        </div>
      </DropdownSection>
    </ResearchDropdownEl>
  )
}

export default ResearchDropdown
