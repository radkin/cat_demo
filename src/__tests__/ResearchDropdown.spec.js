import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import ResearchDropdown, { Counter, dataReducer } from '../DropdownContents/ResearchDropdown';

describe('ResearchDropdown', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<ResearchDropdown />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
