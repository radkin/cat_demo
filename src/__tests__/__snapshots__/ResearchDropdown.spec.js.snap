// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResearchDropdown snapshot renders 1`] = `
<div
  className="sc-AxgMl exIaJC"
>
  <div
    className="sc-AxhUy jUcgjo"
    data-first-dropdown-section={true}
  >
    <ul>
      <li
        className="sc-AxirZ dlCcpm"
      >
        <a
          href="#"
        >
          <div
            className="sc-AxhCb hPcABc"
          />
          Synaptic Transfer
        </a>
      </li>
      <li
        className="sc-AxirZ dlCcpm"
      >
        <a
          href="#"
        >
          <div
            className="sc-AxhCb hPcABc"
          />
          Rapid Skin Growth Toxin
        </a>
      </li>
      <li
        className="sc-AxirZ dlCcpm"
      >
        <a
          href="#"
        >
          <div
            className="sc-AxhCb hPcABc"
          />
          Targeted Genetics
        </a>
      </li>
      <li
        className="sc-AxirZ kBSoHp"
      >
        <a
          href="#"
        >
          <div
            className="sc-AxhCb hPcABc"
          />
          Multidimensional Travel
        </a>
      </li>
    </ul>
  </div>
  <div
    className="sc-AxhUy jUcgjo"
  >
    <div>
      <h3
        className="sc-AxjAm ftcold"
      >
        <div
          className="sc-AxhCb hPcABc"
        />
        Interactive
      </h3>
      <ul
        className="sc-AxiKw jIOAGB"
      >
        <li>
          <a
            href="#"
          >
            Zodiac Paradox ›
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Enhance Physiology and Intellect ›
          </a>
        </li>
        <li>
          <a
            href="#"
          >
            Unforeseeable Changes to Human Civilization ›
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
